<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPasswordDialog</class>
 <widget class="QDialog" name="AddPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="titleInput">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add new password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameInput">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="URLInput">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Url</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="notesInout">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Notes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
